api/src/test/java/org/openmrs/api/handler/RequiredReasonVoidSaveHandlerTest.java

```diff
@@ -77,10 +77,9 @@ public void handle_shouldNotThrowExceptionIfVoidReasonIsNotBlank() {
 	 */
 	@Test
 	public void handle_shouldNotThrowExceptionIfVoidReasonIsNullForUnsupportedTypes() {
-		Person p = Context.getPersonService().getPerson(1);
-		p.setVoided(true);
-		p.setVoidReason(null);
-		p.setVoidReason("voidReason");
-		Context.getPersonService().savePerson(p);
+		PersonAddress pa = Context.getPersonService().getPersonAddressByUuid("3350d0b5-821c-4e5e-ad1d-a9bce331e118");
+		pa.setVoided(true);
+		pa.setVoidReason(null);
+		Context.getPersonService().savePersonAddress(pa);
```

Comment:
This test as written didn't make a lot of sense.  Not sure y it is setting void reason twice.  In any event, the PersonValidator has an explicit check to ensure the void reason is non-null, and based on the name of this test method, the idea is to ensure you can leave void reason null for types other than Patient and Encounter, so I changed this to PersonAddress and things work fine.